docs/developer/pandas/partition_api.rst

```diff
@@ -0,0 +1,75 @@
+Partition API in Modin
+======================
+
+When you are working with a Modin Dataframe, you can unwrap its remote partitions
+to get the raw future objects compatible with the execution engine (e.g. ``ray.ObjectRef`` for Ray).
+In addition to unwrapping of the remote partitions we also provide API to construct Modin DataFrame from them.
+
+API
+---
+
+It is currently supported the following API:
+
+.. automodule:: modin.distributed.dataframe.pandas
+  :noindex:
+  :members: unwrap_partitions
+
+.. automodule:: modin.distributed.dataframe.pandas
+  :noindex:
+  :members: from_partitions
+
+Running an example with Partition API
+-------------------------------------
+
+.. code-block:: python
+
+  import modin.pandas as pd
+  from modin.distributed.dataframe.pandas import unwrap_partitions, from_partitions
+  import numpy as np
+  data = np.random.randint(0, 100, size=(2 ** 10, 2 ** 8))
+  df = pd.DataFrame(data)
+  partitions = unwrap_partitions(df, axis=0)
+  print(partitions)
+  # Also, you can create Modin DataFrame from remote partitions
+  new_df = from_partitions(partitions, axis=0)
+  print(new_df)
+
+Partition IPs Usage
+-------------------
+
+Also, you can use the mentioned above API to get the IPs of the nodes that hold the partitions (``get_ip=True``).
```

Comment:
If we are adding a new API to do this, should we wait to update this document?
```suggestion
You can use the mentioned above APIs to get the IPs of the nodes that hold the partitions (``get_ip=True``).
```

Reply:
Yes, if we add a new API, the doc should be updated. At first we should merge #2643 .